src/instructlab/defaults.py

```diff
@@ -11,11 +11,11 @@
 CONFIG_VERSION = "1.0.0"
 
 # Model families understood by ilab
-MODEL_FAMILIES = {"merlinite", "mixtral"}
+MODEL_FAMILIES = {"granite", "mixtral"}
 
 # Map model names to their family
 MODEL_FAMILY_MAPPINGS = {
-    "granite": "merlinite",
+    "merlinite": "granite",
```

Comment:
can you add `mistral: mixtral` in here? we have an identical copy of this map in SDG I didn't realize there was one here too

Reply:
Yessir will do!